lindera/src/tokenizer.rs

```diff
@@ -92,6 +92,13 @@ impl TokenizerBuilder {
         })
     }
 
+    pub fn from_config(config: TokenizerConfig) -> LinderaResult<Self> {
+        
```

Comment:
This line is causing an error in the workflow format check.
This line is unnecessary.

https://github.com/lindera/lindera/actions/runs/14200465934/job/39954897812?pr=465

Reply:
Alright, thought the format check would automatically format as well.